tools/ci/docker/engine/Dockerfile

```diff
@@ -6,6 +6,7 @@ ARG LOG_LEVEL=info
 ARG RPC_PORT=30001
 ARG DATA_PORT=30002
 ARG YDOC_PORT=5976
+ARG YDOC_HOST="0.0.0.0"
```

Comment:
The `ARG` definition creates an argument that can be configured externally. Why that's necessary?